bazel/clang_tidy/clang_tidy.py

```diff
@@ -22,6 +26,19 @@ def main():
     # so always create it
     Path(fake_output).touch(exist_ok=True)
 
+    runfiles = Runfiles.Create()
+    plugins_dir = runfiles.Rlocation(PLUGINS_DIR)
+    plugin_paths = [
+        Path(runfiles.Rlocation(os.path.join(PLUGINS_DIR, p)))
+        for p in os.listdir(plugins_dir)
+        if ".so" in p
+    ]
```

Comment:
Should we just pass in the list of plugins from the .bzl file so it's explicit?

Reply:
if we're gonna shove everything into one shared object seems reasonable to just hard code, no?

Reply:
done